release build fail

```
In file included from /Volumes/41plus/proton-oss-action/actions-runner/_work/proton/proton/src/Interpreters/TelemetryCollector.cpp:1:
In file included from /Volumes/41plus/proton-oss-action/actions-runner/_work/proton/proton/src/Interpreters/TelemetryCollector.h:3:
/Volumes/41plus/proton-oss-action/actions-runner/_work/proton/proton/src/NativeLog/Base/Concurrent/BlockingQueue.h:60:13: error: use of undeclared identifier 'assert'
   60 |             assert(queue.size() <= max_size);
      |             ^
/Volumes/41plus/proton-oss-action/actions-runner/_work/proton/proton/src/Interpreters/TelemetryCollector.cpp:123:16: note: in instantiation of member function 'nlog::BlockingQueue<std::shared_ptr<DB::TelemetryElement>>::add' requested here
  123 |     if (!queue.add(std::move(element), /*timeout_ms=*/5))
      |                ^
In file included from /Volumes/41plus/proton-oss-action/actions-runner/_work/proton/proton/src/Interpreters/TelemetryCollector.cpp:1:
In file included from /Volumes/41plus/proton-oss-action/actions-runner/_work/proton/proton/src/Interpreters/TelemetryCollector.h:3:
/Volumes/41plus/proton-oss-action/actions-runner/_work/proton/proton/src/NativeLog/Base/Concurrent/BlockingQueue.h:166:13: error: use of undeclared identifier 'assert'
  166 |             assert(queue.empty());
      |             ^
/Volumes/41plus/proton-oss-action/actions-runner/_work/proton/proton/src/Interpreters/TelemetryCollector.cpp:191:24: note: in instantiation of member function 'nlog::BlockingQueue<std::shared_ptr<DB::TelemetryElement>>::drain' requested here
  191 |         auto q = queue.drain(/*timeout_ms=*/10'000);
      |                        ^
2 errors generated.
[7498/9033] Building CXX object src/CMakeFiles/dbms.dir/Interpreters/TranslateQualifiedNamesVisitor.cpp.o
```

--- a/src/NativeLog/Base/Concurrent/BlockingQueue.h
+++ b/src/NativeLog/Base/Concurrent/BlockingQueue.h
@@ -5,6 +5,7 @@
 #include <queue>
 #include <shared_mutex>
 #include <condition_variable>
+#include <cassert>
 
 namespace nlog
 {

--- a/tests/requirements.txt
+++ b/tests/requirements.txt
@@ -1,2 +1,2 @@
-Jinja2==3.1.4
+Jinja2==3.1.5
 termcolor==2.0.1

